src/store/plugins/observables.js

```diff
@@ -19,13 +20,34 @@ export default store => {
       refCountDelay(1000),
     );
 
+  const balance$ = sdk$.pipe(
+    switchMap(sdk => timer(0, 3000).pipe(map(() => sdk))),
+    switchMap(sdk =>
+      sdk
+        ? sdk
+            .balance(store.state.account.publicKey)
+            .catch(error => {
+              if (!isNotFoundError(error)) {
+                handleUnknownError(error);
+              }
+              return 0;
+            })
+            .then(result => convertToAE(result))
+        : Promise.resolve(0),
+    ),
+    multicast(new BehaviorSubject(0)),
```

Comment:
The initial value of the previous balance implementation was persisted in localStorage (as I understand, to don't show 0 at startup). Shouldn't we consider this feature as important and keep it in this implementation?